post-modifier.php

```diff
@@ -30,6 +30,10 @@ private function hooks() {
 		add_action( 'wp_enqueue_scripts', 'al_enqueue_scripts' );
 
 		add_action( 'wp_head', 'al_alter_content_color' );
+
+		add_shortcode( 'al_post_title', 'al_post_title' );
```

Comment:
I'd suggest separating shortcodes from hooks() in the future ( just make a shortcodes() function where you add them all, or add them directly in init )